flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/utils/UserDefinedAggFunctions.java

```diff
@@ -23,13 +23,37 @@
 import java.util.Iterator;
 
 public class UserDefinedAggFunctions {
+    // Accumulator for test requiresOver
+    public static class Accumulator0 extends Tuple2<Long, Integer>{}
 
     // Accumulator for WeightedAvg
     public static class WeightedAvgAccum extends Tuple2<Long, Integer> {
         public long sum = 0;
         public int count = 0;
     }
 
+    // Test for requiresOver
+    public static class OverAgg0 extends AggregateFunction<Long, Accumulator0> {
```

Comment:
please reorder classes that the `AggregateFunction` class follows the accumulator class